### DISCUSSION TITLE
Pro Extensions Changelog

### DISCUSSION THREAD

**Discussion by schnubor** [ueberdosis/tiptap#5570] [General Questions] [answered]
Hey,

I've got a quick question: Is there a public Changelog or something for the latest releases of Pro Extensions (AI, Emojis, etc.)? Since they're in a private repo I usually just upgrade once in a while via npm without actually knowing what the latest version is or what exactly changed. Can I see that somewhere?

Cheers,
Christian

**Comment by nperez0111** [comment, marked as answer]
Hi @schnubor, we need to have a better solution for this, but within the package installed to your node_modules is a CHANGELOG.md file which should give you a summary of what changed. This is not ideal but it is all that we have right now

**Reply by schnubor** [reply]
Thanks Nick! First thing that came to my mind was Changelog section in the respective docs maybe, just as an idea

**Reply by nperez0111** [reply]
Sure, the issue isn't where it should go but coming up with an automation of where to do it every time. It just isn't our priority to be doing this atm
